test/model/message/RawMessage.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 NEM
```

Comment:
This should be (C) Symbol Contributors 2021.

Reply:
I'll update this one, but we would need to update licensing in all the files.

Reply:
````
/*
 * (C) Symbol Contributors 2021
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
````

Is this the new correct licensing? We would need to patch all the files in another pr.